styleguide/source/_patterns/01-molecules/09-search-results/_search-results.scss

```diff
@@ -0,0 +1,130 @@
+.search-result {
```

Comment:
Also, I wonder if this should be called 'best-result' or 'search-result-best', I'm not familiar with the whole system so it might be worth digging into it more, but if the system has a more 'default' search result that differs from the best match, then i think we need to clarify in the name. Might be worth asking Avi